docs/features/software-catalog/descriptor-format.md

```diff
@@ -93,6 +93,41 @@ significance and have reserved purposes and distinct shapes.
 
 See below for details about these fields.
 
+## Substitutions In The Descriptor Format
+
+The descriptor format supports substitutions using `$text`, `$json`, and
+`$yaml`.
+
+Placeholders like `$json: https://example.com/entity.json` are substituted by
+the content of the referenced file. Files can be referenced from any configured
+integration similar to locations by passing an absolute URL. It's also possible
+to reference relative files like `./referenced.yaml` from the same location.
```

Comment:
maybe explicitly mention that they are relative to the folder of the `catalog-info.yaml` file that contains the placeholder

Reply:
Updated it, please take a look if it's expressed well enough.

Reply:
Sure is.